ehrapy/preprocessing/_imputation.py

```diff
@@ -394,72 +353,65 @@ def miss_forest_impute(
     from sklearn.impute import IterativeImputer
 
     try:
-        with Progress(
-            "[progress.description]{task.description}",
-            SpinnerColumn(),
-            refresh_per_second=1500,
-        ) as progress:
-            progress.add_task("[blue]Running MissForest imputation", total=1)
-
-            if settings.n_jobs == 1:  # pragma: no cover
-                logger.warning("The number of jobs is only 1. To decrease the runtime set ep.settings.n_jobs=-1.")
-
-            imp_num = IterativeImputer(
-                estimator=ExtraTreesRegressor(n_estimators=n_estimators, n_jobs=settings.n_jobs),
-                initial_strategy=num_initial_strategy,
-                max_iter=max_iter,
-                random_state=random_state,
-            )
-            # initial strategy here will not be parametrized since only most_frequent will be applied to non numerical data
-            imp_cat = IterativeImputer(
-                estimator=RandomForestClassifier(n_estimators=n_estimators, n_jobs=settings.n_jobs),
-                initial_strategy="most_frequent",
-                max_iter=max_iter,
-                random_state=random_state,
-            )
+        if settings.n_jobs == 1:  # pragma: no cover
```

Comment:
I wonder whether we should remove this warning and just default to -1. There's plenty of algorithms that always default to all available cores.